openfasoc/generators/ldo-gen/blocks/sky130hvl/ldo_domain_insts.txt

```diff
@@ -1,3 +1,26 @@
 cmp1
 pmos_1
 pmos_2
+{pt_array_unit\[0\]}
```

Comment:
So I believe you did not create a model file based on your cdl simulations?

Reply:
Yes. I have not created the model file yet. Once DRC/LVS is done I will be working with simulations.